content/learn/book/rendering/standard-material.md

```diff
@@ -1,11 +1,41 @@
 +++
-title = "TODO"
+title = "Standard Material"
 insert_anchor_links = "right"
 [extra]
 weight = 1
 status = 'hidden'
 +++
 
-Explain PBR, why consistent lighting models are useful
-Explain What a BRDF/material is
-Explain and showcase each StandardMaterial property
+In Bevy, the main `Material` the engine provides is the `StandardMaterial`.
+
+StandardMaterial, like most of the functionality in `bevy_pbr`, implements an idea called "physically based rendering", or PBR.
```

Comment:
Use this link: https://en.wikipedia.org/wiki/Physically_based_rendering

Reply:
Tbh not my favorite wikipedia page. It's kinda generic and not very informative. I'll see if I can find something better.